Public/Src/Tools/Execution.Analyzer/Analyzers.Core/XLGPlusPlus/BXLInvocationAnalyzer.cs

```diff
@@ -79,13 +81,22 @@ internal sealed class BXLInvocationAnalyzer : Analyzer
         /// </summary>
         public string OutputFilePath;
 
-        public BXLInvocationAnalyzer(AnalysisInput input) : base(input) { }
+        private Stopwatch m_stopWatch;
+
+        public BXLInvocationAnalyzer(AnalysisInput input) : base(input)
+        {
+            m_stopWatch = new Stopwatch();
+            m_stopWatch.Start();

```

Comment:
Better start this stopwatch in `Analyze()`.  If you do that, a local variable in `Analyze()` (instead of the `m_stopWatch` field) would suffice